### PR TITLE
Feature implementation of append to rdf skeleton

### PR DESCRIPTION
**Pull request by Shamanou**
Added the new feature.

Also added java script styling.
For now the default behaviour is appending when there is a rdf skeleton (as expected). But when there is no rdf skeleton (new project) then a javascript error is raised internally.

### CODE REVIEW

File: extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js [hunk 1]
```diff
@@ -25,33 +25,38 @@ RdfSkeletonListDialog.prototype._createDialog = function() {
 };
 
 RdfSkeletonListDialog.prototype._constructFooter = function(footer) {
-  var self = this;
+  self = this;
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js]: Why move `self` to the global scope?

File: extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js [hunk 2]
```diff
@@ -25,33 +25,38 @@ RdfSkeletonListDialog.prototype._createDialog = function() {
...
     DialogSystem.dismissUntil(self._level - 1);
   }).appendTo(footer);
 };
 
 RdfSkeletonListDialog.prototype._constructBody = function(body) {
-  var self = this;
-  $(
-          '<p>' + 'Select a RDF skeleton saved earlier to apply to your data.'
-                  + '</p>').appendTo(body);
+  self = this;
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js:61]: Why move `self` to the global scope?

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js [hunk 3]
```diff
@@ -118,7 +116,45 @@ RdfSchemaAlignmentDialog.prototype._constructBody = function(body) {
...
+                        project: theProject.id
+                      }), function(json) {
+                        var operations_old = json.entries.map(function(val) {
+                          return val['operation'];
+                        });
+
+                        if (operations_old.length == 1) {
+                          operations_old = [];
+                        } else {
+                          operations_old = operations_old.slice(-5);
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js:134]: Why only the last 5? Seems like such an arbitrary number.

File: extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
-function RdfSkeletonListDialog(func, schema, project, baseuri) {
+function RdfSkeletonListDialog(func_a, func_b, schema, project, baseuri) {
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-skeleton-list.js:1]: `func_a` and `func_b` are not the most descriptive argument names.

**Review by kburger — APPROVED** [review]
Haven't checked all javascript in depth, but most of it looks good at first glance. Some minor (non-breaking) things that you should take a look at.

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js (hunk 3 above)
Comment (Shamanou) [reply extensions/fairifier-rdf-extension/scripts/rdf-schema-alignment.js:134]: We need to discuss what we want to keep but 5 seems the smallest amount of provenance that still works

**Pull request merged**